src/components/vuestic-components/va-form/VaForm.vue

```diff
@@ -0,0 +1,162 @@
+<script>
+import { ContextPluginMixin, getContextPropValue } from '../../context-test/context-provide/ContextPlugin'
+
+const availableFormTags = ['VaInput', 'VaColorInput', 'VaDatePicker', 'VaCheckbox', 'VaRadio', 'VaSelect', 'VaFileUpload']
```

Comment:
Component should tell form if it's form component, not the other way round.

Reply:
Yes! We need to know form components because need call focus and validate on them.

Reply:
After the implementation of all methods to form-compoents (like a reset, clear, focus, etc.) we need to remove this availableFormElements

Reply:
@Kreezag so are you implying this line is temporary and you agree with what I said? :)